migrations/20170410094328_weighted_wilsons.js

```diff
@@ -0,0 +1,34 @@
+// http://www.evanmiller.org/how-not-to-sort-by-average-rating.html
+exports.up = function(knex, Promise) {
+  return knex.schema.raw(`
+      CREATE FUNCTION wilsons(ups numeric, downs numeric) RETURNS numeric LANGUAGE plpgsql IMMUTABLE AS $$
```

Comment:
Just wondering if the migration is gonna fail because there already exists a function with the same name?